komapper-core/src/main/kotlin/org/komapper/core/dsl/builder/EntityUpdateStatementBuilder.kt

```diff
@@ -35,7 +35,9 @@ class DefaultEntityUpdateStatementBuilder<ENTITY : Any, ID : Any, META : EntityM
             append("update ")
             table(target, support)
             append(" set ")
-            for (p in context.getTargetProperties()) {
+            val targetProperties = context.getTargetProperties()
+            require(targetProperties.isNotEmpty())
```

Comment:
Could you include an error message like the following to clarify the reason for the exception?

> There are no entity properties to specify in the SET clause of the UPDATE statement.